File: api-categoria/handler.py
import json
import boto3

dynamodb = boto3.resource('dynamodb')
categoria_table = dynamodb.Table('Categoria')

def create_categoria(event, context):
    body = json.loads(event['body'])
    tenant_id = body['tenant_id']
    category_id = body['category_id']
    
    # Crear categoría en DynamoDB
    categoria_table.put_item(Item={
        'tenant_id': tenant_id,
        'category_id': category_id,
        'name': body['name']
    })
    
    return {
        'statusCode': 201,
        'body': json.dumps({'message': 'Category created successfully'})
    }

File: api-pedido/handler.js
module.exports.handler = async (event) => {
    const { tenant_id, order_id } = event.pathParameters;
    
    // Lógica para recuperar un pedido de DynamoDB
    return {
      statusCode: 200,
      body: JSON.stringify({ tenant_id, order_id, message: 'Order retrieved successfully' })
    };
  };
  